examples/experimental/dagster-airlift/examples/kitchen-sink/kitchen_sink_tests/integration_tests/test_e2e.py

```diff
@@ -1,14 +1,34 @@
 import os
```

Comment:
would like to see testing for the following behaviors:
- you specify some dag and task mappings using `targeted_by_multiple_tasks`, and then separately use `dag_defs` for other tasks in the same dag
- you specify some dag and task mappings using `targeted_by_multiple_tasks`, and then separately use `dag_defs` to target the _same_ task.